aten/src/ATen/native/Loss.cpp

```diff
@@ -148,4 +148,36 @@ Tensor poisson_nll_loss(const Tensor& input, const Tensor& target, const bool lo
 
     return apply_loss_reduction(loss, reduction);
 }
+
+Tensor l1_loss(const Tensor& input, const Tensor& target, int64_t reduction) {
+  auto loss = input.sub(target).abs_();
+  return apply_loss_reduction(loss, reduction);
+}
+
+Tensor& l1_loss_out(Tensor&result, const Tensor& input, const Tensor& target, int64_t reduction) {
```

Comment:
Where is this function referenced/used?

Reply:
ok, I see there still reamains an enty for `l1_loss.out` in `native_functions.yaml`. Are these special variants of the functions with destination tensor documented somewhere? The new/early entries in 'Loss.cpp' seem not have such special variants. I would prefer if all loss functions would share the same external interface and if the variant with provided destination tensor is important it should be documented somewhere.

Reply:
Please ignore my previous comment. The out-functions are required. The full set of forward, forward_out, backward & backward_out should be implemented.